src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTestBase.cs

```diff
@@ -81,7 +81,7 @@ protected HttpClient CreateHttpClientForRemoteServer(Configuration.Http.RemoteSe
                 wrappedHandler = new VersionCheckerHttpHandler(httpClientHandler, remoteServer.HttpVersion);
             }
 
-            return new HttpClient(wrappedHandler) { DefaultRequestVersion = remoteServer.HttpVersion };
+            return new HttpClient(wrappedHandler) { DefaultRequestVersion = remoteServer.HttpVersion, Timeout = TimeSpan.FromMilliseconds(TestHelper.PassingTestTimeoutMilliseconds) };
```

Comment:
The default timeout is 100 seconds.  This appears to be shortening it to 60 seconds?

If the 100 second timeout was in place and the test was still taking > 20 minutes, changing this timeout to something shorter isn't going to help, is it?